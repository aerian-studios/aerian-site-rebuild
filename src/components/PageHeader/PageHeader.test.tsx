/// <reference types="@types/jest" />
import * as React from "react";

import * as renderer from "react-test-renderer";

import { PageHeader } from "./index";


describe("PageHeader", () => (
    it("renders correctly", () => {
        const tree = renderer
            .create(<PageHeader className="myClass" />
            )
            .toJSON();
        expect(tree).toMatchSnapshot();
    })
));