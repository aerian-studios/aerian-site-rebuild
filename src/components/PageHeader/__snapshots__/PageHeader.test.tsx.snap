// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageHeader renders correctly 1`] = `
<header
  className="block block--full layout-grid component myClass"
  id="page-header"
  style={Object {}}
/>
`;
